src/components/ShoppingList/PopulatedList/PopulatedList.js

```diff
@@ -143,58 +145,103 @@ const PopulatedList = () => {
     }
   }
 
+  function formatTime(seconds) {
+    const months = {
+      0: "Jan.",
+      1: "Feb.",
+      2: "March",
+      3: "April",
+      4: "May",
+      5: "June",
+      6: "July",
+      7: "Aug.",
+      8: "Sept.",
+      9: "Oct.",
+      10: "Nov.",
+      11: "Dec.",
+    };
+    const timestamp = new Date(seconds * 1000);
+    return `${
+      months[timestamp.getMonth()]
+    } ${timestamp.getDate()}, ${timestamp.getFullYear()}`;
+  }
+
```

Comment:
Awesome job with this function to format our date!

Reply:
Agreed!! Great work @thekidnamedkd 😄